examples/express-es7/package.json

```diff
@@ -3,45 +3,26 @@
   "version": "0.8.0",
   "description": "Objection.js express ES7 example",
   "main": "app.js",
-
   "scripts": {
     "migrate": "knex migrate:latest",
     "build": "babel src -d build",
     "start": "npm run build && npm run migrate && node build/app"
   },
-
   "babel": {
     "presets": [
-      "stage-0"
-    ],
-    "plugins": [
-      "check-es2015-constants",
-      "transform-es2015-arrow-functions",
-      "transform-es2015-block-scoped-functions",
-      "transform-es2015-block-scoping",
-      "transform-es2015-computed-properties",
-      "transform-es2015-destructuring",
-      "transform-es2015-duplicate-keys",
-      "transform-class-properties",
-      "transform-es2015-for-of",
-      "transform-es2015-function-name",
-      "transform-es2015-literals",
-      "transform-es2015-modules-commonjs",
-      "transform-es2015-object-super",
-      "transform-es2015-parameters",
-      "transform-es2015-shorthand-properties",
-      "transform-es2015-spread",
-      "transform-es2015-sticky-regex",
-      "transform-es2015-template-literals",
-      "transform-es2015-typeof-symbol",
-      "transform-es2015-unicode-regex",
-      "transform-async-to-generator"
+      [
+        "env",
+        {
+          "targets": {
+            "node": "current"
+          }
+        }
+      ],
+      "stage-1"
```

Comment:
The original `package.json` used `stage-0` but @koskimas used `stage-1` in his example in #572 so I changed it here.